client/header/activity-panel/panels/inbox/inbox-panel.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import { sprintf } from '@wordpress/i18n';
+import { Text } from '@woocommerce/experimental';
+import { recordEvent } from '@woocommerce/tracks';
+import { AbbreviatedCard } from '@woocommerce/components';
+
+/**
+ * Internal dependencies
+ */
+import './inbox.scss';
+import NotesPanel from '../../../../inbox-panel';
+import { cards } from './cards';
```

Comment:
Do we need to represent the cards as data? Could we wire this component up to state instead and have it decide which `AbbreviatedCard`s to render explicitly? I'm just not seeing the benefit to representing the cards and unread counts as objects that have to be matched up later in `getCardByName()`, since we aren't applying a filter or anything here.

Reply:
Good idea! I changed it in the commit 8a9bdac and added tests in the commit fa25dc3